sentry-android-fragment/src/main/java/io/sentry/android/fragment/FragmentLifecycleIntegration.kt

```diff
@@ -0,0 +1,66 @@
+package io.sentry.android.fragment
+
+import android.app.Activity
+import android.app.Application
+import android.app.Application.ActivityLifecycleCallbacks
+import android.os.Bundle
+import androidx.fragment.app.FragmentActivity
+import io.sentry.IHub
+import io.sentry.ILogger
+import io.sentry.Integration
+import io.sentry.SentryLevel.DEBUG
+import io.sentry.SentryOptions
+import java.io.Closeable
+
+class FragmentLifecycleIntegration(
+    private val application: Application,
+    private val logger: ILogger
```

Comment:
the logger can be read from `options.logger`, so a `lateinit` var would help too

Reply:
Right! Fixed in 09f1c6305cba5507e15f04eec13f7df46ab7ac3d